src/components/pages/Home/index.vue

```diff
@@ -30,12 +30,14 @@ export default {
   name: 'home',
   data () {
     return {
-      msg: 'Welcome to Your Vue.js App'
+      code: '',
+      errorMsg: null
     }
   },
   methods: {
     searchByCode () {
       this.$validator.validateAll().then(res => {
+        this.errorMsg = this.errors
```

Comment:
Mmm.. I think this is quite unnecessary since you have all the errors in that errors attribute. Just use that object instead of "copying" that and use that new object.